### PR TITLE
[R-package] fixed R Dataset aliases

### PR DESCRIPTION
**Pull request by StrikerRUS**
Refer to #2594.

### CODE REVIEW

File: R-package/R/aliases.R [hunk 1]
```diff
@@ -51,12 +46,19 @@
...
+        )
         , "label_column" = c(
             "label_column"
             , "label"
         )
         , "max_bin" = "max_bin"
         , "max_bin_by_feature" = "max_bin_by_feature"
+        , "min_data_in_bin" = "min_data_in_bin"
         , "pre_partition" = c(
             "pre_parition"
```
Comment (jameslamb) [inline R-package/R/aliases.R]:
```suggestion
            "pre_partition"
```

Comment (jameslamb) [reply R-package/R/aliases.R]: ^ noticed this one 😬

**Comment by StrikerRUS** [comment]
@jameslamb 
> Honestly can't believe I didn't make more mistakes, manually entering these from the docs

I even cannot remember how many times I checked that for Python! Maybe 5?.. 😃

That's why we have #2633.

**Comment by jameslamb** [comment]
This looks good to me! Feel free to merge once the build is green

**Pull request merged**